src/content/blog/astro-220.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Astro 2.2"
+description: "CDN support for assets • image() schema helper API change • Much more!"
+publishDate: "April 5, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-220/blog-hero-2_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-220/og-image-2_2-release.jpg"
+lang: "en"
+---
+
+Astro 2.2 is out! Included in this release are:
+
+- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will loads from there.
+- **[image() schema helper API change](#image-schema-api-change)**: (experimental) The image() helper has been moved to be inline with the schema definition.
+
+Additionally Astro 2.2 includes **8 patch changes** and was released alongside **8 integrations** with patch releases.
+
+Run `npm i astro@latest` to upgrade to the latest version.
+
+## CDN support
+
+In 2.2 you can now configure Astro to point to a CDN in order to load your assets. In your Astro config:
+
+```js
+import { defineConfig } from "astro/config"
+
+export default defineConfig({
+	build: {
+		assetsPrefix: "https://cdn.example.com",
+	},
+})
+```
+
+After you've run the build (`astro build`) your HTML (both in SSG and SSR modes) will include links that include the prefix for any assets; scripts and stylesheets.
+
+```html
+<link rel="stylesheet" href="https://cdn.example.com/_astro/global.css" />
+```
+
+The assets are still built to your `dist/` folder, like always. You will need to configure your deployment scripts to deploy `dist/_astro/` to your CDN.
+
+## image() schema API change
+
+In 2.2 the `image()` schema helper in the experimental assets API has been moved to a callback where the schema is defined.
+
+```js
+import { defineCollection, z } from "astro:content"
+
+const blog = defineCollection({
+	schema: ({ image }) =>
+		z.object({
+			image: image().refine((img) => img.width >= 200, {
+				message: "image too small",
+			}),
+		}),
+})
+```
+
+Previously `image()` was imported directly from `'astro:content'`. This created bugs where Astro was not able to correctly resolve relative image assets defined in the frontmatter.
+
+Providing the helper as a callback aligns with how Zod suggests doing contextual references. This new callback based schema assignment is currently only available behind the assets flag. If you are not using the image() helper you can continue to assign the schemas directly to the `schema` property on the collection.
```

Comment:
```suggestion
Previously, `image()` was imported directly from `'astro:content'`. However, this created bugs where Astro was not able to correctly resolve relative image assets defined in the frontmatter. Providing the helper as a callback aligns with how Zod suggests doing contextual references. 

This new callback based schema assignment is currently only available behind the `--assets` flag. If you are not using the `image()` helper, you can continue to assign the schemas directly to the `schema` property on the collection.
```
I feel like this splits the paragraphs up better: first paragraph for "mistakes were made" and then second paragraph starting here doesn't hide that this only affects the experimental branch, and provides guidance for what that means to the user.